tools/wheel_resolver/__init__.py

```diff
@@ -60,14 +61,23 @@
     multiple=True,
     help="The ABI identifier, for example cp310 or abi3",
 )
+@click.option(
+    "--prereleases",
+    default=False,
+    metavar="PRERELEASES",
+    show_default=True,
+    multiple=False,
+    help="Whether prereleased wheels should also be downloaded",
+)
 @click_log.simple_verbosity_option(_LOGGER)
 def main(
     url: typing.List[str],
-    package_name: typing.Optional[str],
+    package_name: str,
     package_version: typing.Optional[str],
     interpreter: typing.Tuple[str, ...],
     platform: typing.Tuple[str, ...],
     abi: typing.Tuple[str, ...],
+    prereleases: bool = False,
```

Comment:
I assume that adding this will be enough for click to turn it into a flag?

Reply:
Yes given the `@click.option` decorator for `--prereleases` above